sdk/synapse/synapse-access-control-rest/src/longRunningHelper.ts

```diff
@@ -0,0 +1,51 @@
+import { Client, HttpResponse } from "@azure-rest/core-client";
+import { PollOperationState, PollerLike } from "@azure/core-lro";
+import { createGetLROState, LROPoller } from "./lro";
+import { LROPollerOptions } from "./lro/lroPoller";
+import { LRO, LROBody, LROState } from "./lro/models";
+
+/**
+ * Note: the contents of this file will live in @azure-rest/core-client
+ */
+export function getLongRunningPoller<TResult extends HttpResponse>(
```

Comment:
@deyaaeldeen - fyi

Reply:
Looking great!

> Note: the contents of this file will live in @azure-rest/core-client

Are you sure you want `@azure-rest/core-client` to take a dependency on `@azure/core-lro`? I have same dilemma in `@azure/core-client` so I decided to keep it autogenerated for now.